packages/api/src/clickhouse/index.ts

```diff
@@ -2392,8 +2402,12 @@ SELECT *
 FROM sessions 
 WHERE sessions.sessionId IN (
     SELECT sessionIdsWithRecordings.sessionId FROM sessionIdsWithRecordings
+  ) OR sessions.sessionId IN (
+    SELECT sessionIdsWithUserActivity.sessionId FROM sessionIdsWithUserActivity
   )`,
```

Comment:
I haven't benchmarked this atm but I know a prior approach with joins had performance issues. I believe the current query formulation is different from the joins and should be more efficient as I'm asking the DB to effectively do a set intersection, rather than matching row x row lookups (which I'm not confident the CH query optimizer will handle correctly given prior experience)

Reply:
Okay cool thanks for the color - I assumed as much - i know the CH format allows for some diff optimizations vs say postgres but am not very familiar what what those are yet obviously. That makes sense to me though 👍